templates/CRM/Event/Form/Registration/ThankYou.tpl

```diff
@@ -81,7 +81,7 @@
                 {$event.fee_label}
             </div>
             {if $lineItem}
-                {include file="CRM/Price/Page/LineItem.tpl" context="Event"}
+                {include file="CRM/Price/Page/LineItem.tpl" context="Event" displayLineItemFinancialType=false getTaxDetails=$totalTaxAmount hookDiscount=''}
```

Comment:
For hookDiscount this seems like it might overwrite it if it had been set here https://github.com/civicrm/civicrm-core/blob/bb103e445e143c64454bc9a1fa93bacec5648bd0/CRM/Event/Form/Registration/ThankYou.php#L118, but having said that, there's nothing in universe that does that except the old civi sample extension, and the hook eventDiscount docs don't tell you how to use it, and this strategy is used elsewhere (although might be wrong). At some point maybe we should officially deprecate it.